Plugins/RPRPlugin/Source/RPRPlugin/Private/Renderer/RPRRendererWorker.cpp

```diff
@@ -180,11 +181,12 @@ bool	FRPRRendererWorker::ResizeFramebuffer(uint32 width, uint32 height)
 		return false;
 
 	m_PreRenderLock.Lock();
+
 	m_Width = width;
 	m_Height = height;
 
 	m_Resize = true;
-	m_RenderData.SetNumZeroed(m_Width * m_Height * sizeof(float));
+	m_ExpectedRenderDataSize = m_Width * m_Height * sizeof(float) * (sizeof(float) / sizeof(uint8));
```

Comment:
sizeof(uint8) should be 1, does it really needed?

Reply:
I think, better to leave sizeof(unit8). 

uint8 is datatype of buffer (buffer of uint8). Just to be sure, that this formula is what we need for correct size calculation